dash/dash.py

```diff
@@ -1338,14 +1338,19 @@ def enable_dev_tools(
             _reload = self._hot_reload
             _reload.hash = generate_hash()
 
+            # find_loader should return None on __main__ but doesn't
+            # on some python versions https://bugs.python.org/issue14710
+            packages = [
+                pkgutil.find_loader(x)
+                for x in list(ComponentRegistry.registry) + ["dash_renderer"]
+                if x not in ("__main__")
```

Comment:
```suggestion
                if x != "__main__"
```

Or `if x not in ("__main__",)` but we might as well do simple `!=` until we find more cases we need to exclude.

Reply:
thanks!